### ISSUE TITLE
Add Page for listing broken and damaged appliances

### ISSUE THREAD

**Issue by SepsiLaszlo** [kir-dev/konyha#62]

**Comment by triszt4n** [comment]
Is this for the KB people? I mean do we list ALL the broken and damaged appliances or only per user?

**Comment by SepsiLaszlo** [comment]
Yes, this was reguested by the KB. We should list all the broken and damaged appliances in the House. 
This is aimed to help the Maintainers of the Hostel find the appliances that are need to be fixed, so we can filter out appliances that are not basic. We should discuss the grouping and the ordering of the appliances later, now we can implement this feature without specific implementation for these two attribute.
